examples/categorized-menu/app.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
```

Comment:
This example seems to be the last outstanding feature since rmcauley's changes.  Let me know if you have any more suggestions, @CMTegner 

I guess the example really comes down to a question of why "Custom Menu" was created in the first place.  Either to demo unselectable headers (we can replace it entirely) or to demonstrate an example of custom rendering (probably worth keeping)